### PR TITLE
feature/LLC-001 - Deleting blank spaces

### PR DESCRIPTION
**Pull request by Rostanxd**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -17,12 +17,8 @@
...
--
 - [Alex Bonilla](https://twitter.com/alexferbonilla)
 - [Steven Guamán](https://www.linkedin.com/in/steven-guam%C3%A1n-figueroa-ba3ba816b/)
 - [Julio Armijos](https://twitter.com/julioarmijos)
 - Juan Marin
-- 
-- [Daniela Condor](https://twitter.com/dani_cndr?s=08)
--
--
+- [Daniela Condor](https://twitter.com/dani_cndr?s=08)
```
Comment (micha3l999) [inline docs/index.md:30]: This line should be in line 30

File: docs/index.md [hunk 2]
```diff
@@ -17,12 +17,8 @@
 ## Team List
 
 - [Carlos Carvajal](https://twitter.com/_CarlosXavier98)
--
 - [Alex Bonilla](https://twitter.com/alexferbonilla)
 - [Steven Guamán](https://www.linkedin.com/in/steven-guam%C3%A1n-figueroa-ba3ba816b/)
 - [Julio Armijos](https://twitter.com/julioarmijos)
 - Juan Marin
```
Comment (pichiloor) [inline docs/index.md]: Si Juan Marin no tiene perfil, borremos la línea

File: docs/index.md [hunk 3]
```diff
@@ -17,12 +17,8 @@
 ## Team List
 
 - [Carlos Carvajal](https://twitter.com/_CarlosXavier98)
--
 - [Alex Bonilla](https://twitter.com/alexferbonilla)
 - [Steven Guamán](https://www.linkedin.com/in/steven-guam%C3%A1n-figueroa-ba3ba816b/)
```
Comment (YpordeDx) [inline docs/index.md:21]: Use a shortener option like Bit.ly

**Review by YpordeDx — COMMENTED** [review]
Check de review on index.md file

File: docs/index.md (hunk 1 above)
Comment (Rostanxd) [reply docs/index.md:30]: ok, I'll fix it! ty

File: docs/index.md (hunk 2 above)
Comment (Rostanxd) [reply docs/index.md]: ok, good catch!

**Review by micha3l999 — APPROVED** [review]
good job!

**Review by YpordeDx — APPROVED** [review]
OK, go ahead

**Review by pichiloor — APPROVED** [review]
Ok, todo bien ahora.

**Pull request merged**